tools/shims/src/TsLibShims.ts

```diff
@@ -1,11 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import { getGlobal, objAssign, objCreate, objDefineProp, objHasOwnProperty, throwTypeError } from "@nevware21/ts-utils";
```

Comment:
Re-Using the ts-utils versions to avoid the repeated code -- provides smaller final code.